packages/web-components/src/link/link.stories.ts

```diff
@@ -1,71 +1,143 @@
 import { html } from '@microsoft/fast-element';
-import type { Args, Meta } from '@storybook/html';
-import { renderComponent } from '../helpers.stories.js';
+import { type NewMeta as Meta, renderComponent, type StoryArgs, type StoryObj } from '../helpers.stories.js';
+import { AnchorTarget } from '../anchor-button/anchor-button.options.js';
 import type { Link as FluentLink } from './link.js';
 import { LinkAppearance } from './link.options.js';
 
-type LinkStoryArgs = Args & FluentLink;
-type LinkStoryMeta = Meta<LinkStoryArgs>;
+type Story = StoryObj<FluentLink>;
 
-const storyTemplate = html<LinkStoryArgs>`
-  <fluent-link href="${x => x.href}" appearance="${x => x.appearance}">${x => x.content}</fluent-link>
+const storyTemplate = html<StoryArgs<FluentLink>>`
+  <fluent-link
+    href="${story => story.href}"
+    hreflang="${story => story.hreflang}"
+    ping="${story => story.ping}"
+    referrerpolicy="${story => story.referrerpolicy}"
+    rel="${story => story.rel}"
+    type="${story => story.type}"
+    target="${story => story.target}"
+    inline="${story => story.inline}"
+    appearance="${story => story.appearance}"
+  >
+    ${story => story.slottedContent?.()}
+  </fluent-link>
 `;
 
 export default {
   title: 'Components/Link',
+  render: renderComponent(storyTemplate),
   args: {
-    content: 'Link',
+    slottedContent: () => 'Link',
     href: '#',
-    disabled: false,
-    disabledFocusable: false,
   },
   argTypes: {
-    appearance: {
-      options: Object.values(LinkAppearance),
-      control: {
-        type: 'select',
+    href: {
+      control: 'text',
+      description: 'The href of the anchor.',
+      name: 'href',
+      table: { category: 'attributes', type: { summary: 'string' } },
+    },
+    hreflang: {
+      control: 'text',
+      description: 'Hints at the language of the referenced resource.',
+      name: 'hreflang',
+      table: { category: 'attributes', type: { summary: 'string' } },
+    },
+    ping: {
+      control: 'text',
+      description: 'The ping attribute.',
+      name: 'ping',
+      table: { category: 'attributes', type: { summary: 'string' } },
+    },
```

Comment:
I realize it's probably in the spec but I think it's disabled in Firefox and probably not recommended to use. According to https://caniuse.com/ping

> While still in the WHATWG specification, this feature was removed from the W3C HTML5 specification in 2010

No action necessary on this PR but may need a followup issue to remove.

Reply:
I'll remove